presentation-api/controlling-ua/defaultRequest_error-manual.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<meta charset="utf-8">
+<title>Presentation API, testing to start a new presentation with an default request setup. (error - manual)</title>
+<link rel="author" title="Marius Wessel" href="http://www.fokus.fraunhofer.de">
+<link rel="help" href="http://w3c.github.io/presentation-api/#dfn-controlling-user-agent">
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+
+<p>Click the button below and abort the selection for an device (close the dialog), to start the manual test.</p>
```

Comment:
Ah, okay. Just looked at startNewPresentation and there we've added an error test case.

So, delete the entire file?